util/helpfulIO.py

```diff
@@ -8,19 +7,18 @@
 
 import constants
 
-LimitSwitch = Callable[[], bool]
-
 
 class Falcon:  # represents either a simulated motor or a real Falcon 500
     def __init__(
         self,
         name: str,
         realId: int,
         simId: int,
-        PGain: int = 1,
-        IGain: int = 0,
-        DGain: int = 0,
+        PGain: float = 0.01,
```

Comment:
Why so low?

Reply:
default should be a safe number - P gain of 1 with little strain on the motor causes it to go back and forth

Reply:
Alright, my only issue with this is it makes it a bit tougher cuz then we basically need to specify a special P gain for every motor: we should probably do that anyway but for me I don't generally know what the best ones are going to be.